src/app/shared/navigation/store/epics.ts

```diff
@@ -22,8 +22,8 @@ export const appNavigationEpics: Epics = {
   ),
 
   interruptedNavigationSaving: (action$) => action$.pipe(
-    ofType(AuthActions.unauthorize),
-    tap(({ payload }) => {
+    filter(AuthActions.unauthorize.match),
+    tap<PayloadAction<unauthorizePayload>>(({ payload }) => {
```

Comment:
If we will create actions with `definceAction`, we will be able to remove this generic type